resolver/context.go

```diff
@@ -126,7 +126,7 @@ func (c *context) argIndex() int {
 	return c.argIdx
 }
 
-func (c *context) addError(e *LocationError) {
```

Comment:
For https://github.com/mercari/grpc-federation/pull/15/files#diff-732f365bd38694dea48332c13814847ee19a76bebc5f3d2c87733f1a4ee915ffR99